src/services/mail.ts

```diff
@@ -0,0 +1,63 @@
+import { InternalServerError } from "@/types/exceptions";
+import nodemailer, { Transporter } from "nodemailer";
+
+class EmailService {
+  private transporter: Transporter;
+
+  constructor() {
+    this.transporter = nodemailer.createTransport({
+      host: "smtp.gmail.com",
+      port: 465,
+      secure: true,
+      auth: {
+        user: process.env.MAIL_USER,
+        pass: process.env.MAIL_PASS,
+      },
+    });
+  }
+
+  async sendEmail(to: string, subject: string, html: string): Promise<boolean> {
```

Comment:
```suggestion
  static async sendEmail(to: string, subject: string, html: string): Promise<boolean> {
```

Make all these methods static!